src/pages/Facility/settings/organizations/components/FacilityOrganizationSelector.tsx

```
@@ -406,41 +421,17 @@ export default function FacilityOrganizationSelector(
-                  </Drawer>
-                </>
-              ) : (
-                <Popover open={open} onOpenChange={handleOpenChange}>
-                  <PopoverTrigger asChild>
+            {isMobile ? (
+              <>
+                <Drawer open={open} onOpenChange={setOpen}>
+                  <DrawerTrigger asChild>
                     <Button
                       variant="outline"
                       role="combobox"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<select>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)